frontend/src/components/ui/CustomSelect.tsx

```diff
@@ -0,0 +1,47 @@
+import {Box, Chip, FormControl, InputLabel, MenuItem, OutlinedInput, Select, SelectChangeEvent} from "@mui/material";
+
+type CustomSelectProps = {
+    selectedValue: string[]
+    data: string[]
+    label:string
+    fieldName:string
+    handleSelectChange(event: SelectChangeEvent<string[]>): void;
+}
+export default function CustomSelect(props: CustomSelectProps) {
+
+    function handleSelectChange(event: SelectChangeEvent<string[]>) {
+        props.handleSelectChange(event)
+    }
+
+    return (
+        <FormControl fullWidth={true}
+                     margin={"dense"}>
+
+            <InputLabel>{props.label}</InputLabel>
+
+            <Select
+                name={props.fieldName}
+                multiple={true}
+                input={<OutlinedInput label={props.label}/>}
+                value={props.selectedValue}
+                onChange={handleSelectChange}
+                renderValue={(selected: string[]) => (
+                    <Box sx={{display: 'flex', flexWrap: 'wrap', gap: 0.5}}>
```

Comment:
I assume `Box` doesn't have the option to set the css properties like you did in `Select`?